app/cni/pkg/cni/utils.go

```diff
@@ -0,0 +1,84 @@
+package cni
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+// PidOf Find process(es) with a specified name (string match)
+// copied from https://github.com/kubernetes/kubernetes/blob/v1.24.3/pkg/util/procfs/procfs_linux.go#L99
+// with small modifications
+// and return their pid(s)
+func pidOf(name string) ([]int, error) {
+	if len(name) == 0 {
+		return []int{}, errors.New("name should not be empty")
+	}
+	return getPids(name), nil
+}
+
+// we don't need regex so this is changed to "string"
+func getPids(name string) []int {
+	pids := []int{}
```

Comment:
maybe
```suggestion
	var pids []int
```
?

Reply:
that's the release branch, so let's move the discussion about it in different PR